packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -2233,7 +2233,7 @@ it('should handle table resize only when some columns were resized', () => {
       triggerResize = onResize;
       return [
         jest.fn(),
-        ({ disconnect: jest.fn() } as unknown) as ResizeObserver,
+        { disconnect: jest.fn() } as unknown as ResizeObserver,
```

Comment:
No idea why it made random formatter changes.

Reply:
what have you done with your formatter :D

Reply:
Nothing, I saw similar changes in Jon's PR. Our rules must have changed with monorepo

Reply:
Changed or removed?

Reply:
idk, ask Mayank

Reply:
@mayank99 any ideas, what could be happening here?

Reply:
Prettier version must have changed in the monorepo. I will create a PR running `prettier` on all files.

Reply:
It was because of multiple conflicting versions of `prettier`. I've created #651 to fix it.

@bentleyvk Can you verify that it fixes the issue in your PR?

Reply:
I copied `yarn.lock`, ran `yarn install` and tried to format `Table.tsx` but nothing happened, I also tried to commit the file but still it remain the same - with changed formatting.

Reply:
Ok, had to refresh VS Code, now it formats. But strange why pre-commit hook didn't format...

Reply:
Can I just merge before waiting for prettier fix? Because there is a user that is waiting for my fixes.

Reply:
I fixed and pushed the formatting in https://github.com/iTwin/iTwinUI-react/pull/648/commits/d6f33ee2f6ba2cfea8cbd0153195e81834682ba1. Should be good to merge now.